Update wechatpay-apiv3.postman_collection.json

Memory-9527 opened the pull request:
补充了微信支付普通商户已知公开接口，每个接口请求预置了请求参数示例与请求成功返回的参数示例,修改原脚本变量为常用叫法，无其他修改部分： merchantId->mchid
merchantSerialNo->merchant_serial_no
merchantPrivateKey->apiclient_key.pem
预置请求参数需要配置全局变量
endpoint	必填	微信支付接口域名	固定值：https://api.mch.weixin.qq.com
 appid	选填	用于微信支付接口请求中的APPID	如不配置全局变量，请求时需要将参数中的变量替换为实际APPID值，填写APPID需要与填写的mchid有绑定关系
 openid	选填	用于微信支付接口请求中的openid	如不配置全局变量，请求时需要将参数中的变量openid替换为实际openid值

Memory-9527 committed:
Update wechatpay-apiv3.postman_collection.json

补充了微信支付普通商户已知公开接口，每个接口请求预置了请求参数示例与请求成功返回的参数示例,修改原脚本变量为常用叫法，无其他修改部分：
merchantId->mcid
merchantSerialNo->merchant_serial_no
merchantPrivateKey->apiclient_key.pem
预置请求参数需要配置全局变量
endpoint	必填	微信支付接口域名	固定值：https://api.mch.weixin.qq.com
appid	选填	用于微信支付接口请求中的APPID	如不配置全局变量，请求时需要将参数中的变量openid替换为实际openid值，填写APPID需要与填写的mchid有绑定关系
openid	选填	用于微信支付接口请求中的openid	如不配置全局变量，请求时需要将参数中的变量openid替换为实际openid值

xy-peng commented:
感谢贡献！

> endpoint 必填 微信支付接口域名 固定值：https://api.mch.weixin.qq.com/

参考 [OpenAPI](https://swagger.io/docs/specification/api-host-and-base-path/) ，

```plain
https://api.example.com/v1/users?role=admin&status=active
\________________________/\____/ \______________________/
         server URL       endpoint    query parameters
                            path
```
`endpoint` 应该改成 `server_url`

xy-peng commented:
另外，在 postman 也发起 pull request ？

Memory-9527 commented:
> 感谢贡献！
> 
> > endpoint 必填 微信支付接口域名 固定值：https://api.mch.weixin.qq.com/
> 
> 参考 [OpenAPI](https://swagger.io/docs/specification/api-host-and-base-path/) ，
> 
> ```
> https://api.example.com/v1/users?role=admin&status=active
> \________________________/\____/ \______________________/
>          server URL       endpoint    query parameters
>                             path
> ```
> 
> `endpoint` 应该改成 `server_url`

稍后我修正后重新推送

Memory-9527 committed:
Update wechatpay-apiv3.postman_collection.json

补充了微信支付普通商户已知公开接口，每个接口请求预置了请求参数示例与请求成功返回的参数示例,修改原脚本变量为常用叫法，无其他修改部分：
merchantId->mcid
merchantSerialNo->merchant_serial_no
merchantPrivateKey->apiclient_key.pem
预置请求参数需要配置全局变量
endpoint	必填	微信支付接口域名	固定值：https://api.mch.weixin.qq.com
appid	选填	用于微信支付接口请求中的APPID	如不配置全局变量，请求时需要将参数中的变量openid替换为实际openid值，填写APPID需要与填写的mchid有绑定关系
openid	选填	用于微信支付接口请求中的openid	如不配置全局变量，请求时需要将参数中的变量openid替换为实际openid值

修改endpoint变量名为server_url

Memory-9527 commented:
> 另外，在 postman 也发起 pull request ？
只能同步，endpoint 已修正为 server_url
<img width="1260" alt="image" src="https://user-images.githubusercontent.com/30432849/200219326-b5ae284d-fdb4-4147-a5a2-fd7fae686233.png">

TheNorthMemory commented on wechatpay-apiv3.postman_collection.json:
```diff
@@ -123,160 +113,8699 @@
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{server_url}}/v3/bill/tradebill?bill_date=2022-12-31&bill_type=ALL",
```
这里建议把`bill_date`的值设成变量，默认值可以是当前`北京时间日期` - 1，如果是UTC/GMT时间，不work的

TheNorthMemory commented on wechatpay-apiv3.postman_collection.json:
```diff
@@ -123,160 +113,8699 @@
+									"query": [
+										{
+											"key": "bill_date",
+											"value": "2022-12-31",
```
值需要是北京时间

TheNorthMemory commented on wechatpay-apiv3.postman_collection.json:
```diff
@@ -123,160 +113,8699 @@
+										"method": "GET",
+										"header": [],
+										"url": {
+											"raw": "{{server_url}}/v3/bill/tradebill?bill_date=2020-12-31&bill_type=ALL",
```
`bill_date`的值，建议用变量

TheNorthMemory commented on wechatpay-apiv3.postman_collection.json:
```diff
@@ -123,160 +113,8699 @@
+										"method": "POST",
+										"header": [],
+										"url": {
+											"raw": "{{domain}}/v3/pay/transactions/app",
```
`{{domain}}` -> `{{server_url}}`

TheNorthMemory reviewed (commented):
great job!

TheNorthMemory commented on wechatpay-apiv3.postman_collection.json:
```diff
@@ -123,160 +113,8699 @@
+							"name": "服务人员信息更新",
+							"request": {
+								"method": "PATCH",
+								"header": [],
```
这里需要上送 `Wechatpay-Serial` 平台证书序列号，官方文档缺

TheNorthMemory commented on wechatpay-apiv3.postman_collection.json:
```diff
@@ -123,160 +113,8699 @@
+										"header": [],
+										"body": {
+											"mode": "raw",
+		
```
`key` & `value` 同时为`null` 了，应该不需要

TheNorthMemory replied:
`POST` 请求，以及可选需要 `Wechatpay-Serial` 头（当有 `name` 加密入参时)

TheNorthMemory replied:
`POST` 请求

Memory-9527 committed:
Update wechatpay-apiv3.postman_collection.json

修正部分错误，增加部分接口提示

Memory-9527 committed:
Update wechatpay-apiv3.postman_collection.json

修正添加分账接收方请求方式

Memory-9527 committed:
Update wechatpay-apiv3.postman_collection.json

修正添加分账接收方请求方式

Memory-9527 committed:
Update wechatpay-apiv3.postman_collection.json

修正返回示例与请求方法不一致的错误

TheNorthMemory reviewed (approved):
真棒！

TheNorthMemory commented:
类别|产品|服务|`URL(pathname)`
---|---|---|---
| | 微信支付平台证书 | 获取微信支付平台证书列表 |  /v3/certificates
基础支付 | JSAPI支付 | JSAPI下单 |  /v3/pay/transactions/jsapi
基础支付 | JSAPI支付 | 查询订单_商户订单号查询 |  /v3/pay/transactions/out-trade-no/{{将此处替换为要查询的商户订单号}}?mchid={{mchid}}
基础支付 | JSAPI支付 | 查询订单_微信支付订单号查询 |  /v3/pay/transactions/id/{{将此处替换为要查询的微信支付单号}}?mchid={{mchid}}
基础支付 | JSAPI支付 | 关闭订单 |  /v3/pay/transactions/out-trade-no/{{此处替换为要关闭的商家订单号}}/close
基础支付 | JSAPI支付 | 申请退款 |  /v3/refund/domestic/refunds
基础支付 | JSAPI支付 | 查询单笔退款 |  /v3/refund/domestic/refunds/{{此处替换为要查询的商户自己生成的退款单号}}
基础支付 | JSAPI支付 | 申请交易账单 |  /v3/bill/tradebill?bill_date=2022-12-31&bill_type=ALL
基础支付 | JSAPI支付 | 申请资金账单 |  /v3/bill/fundflowbill?bill_date=2022-12-31
基础支付 | JSAPI支付 | 下载账单 |  /v3/billdownload/file?token=6XIv5TUPto7pByrTQKhd6kwvyKLG2uY2wMMR8cNXqaA_Cv_isgaUtBzp4QtiozLO
基础支付 | APP支付 | APP下单 |  /v3/pay/transactions/app
基础支付 | APP支付 | 查询订单_商户订单号查询 |  /v3/pay/transactions/out-trade-no/{{将此处替换为要查询的商户订单号}}?mchid={{mchid}}
基础支付 | APP支付 | 查询订单_微信支付订单号查询 |  /v3/pay/transactions/id/{{将此处替换为要查询的微信支付单号}}?mchid={{mchid}}
基础支付 | APP支付 | 关闭订单 |  /v3/pay/transactions/out-trade-no/{{此处替换为要关闭的商家订单号}}/close
基础支付 | APP支付 | 申请退款 |  /v3/refund/domestic/refunds
基础支付 | APP支付 | 查询单笔退款 |  /v3/refund/domestic/refunds/{{此处替换为要查询的商户自己生成的退款单号}}
基础支付 | APP支付 | 申请交易账单 |  /v3/bill/tradebill?bill_date=2022-12-31&bill_type=ALL
基础支付 | APP支付 | 申请资金账单 |  /v3/bill/fundflowbill?bill_date=2022-12-31
基础支付 | APP支付 | 下载账单 |  /v3/billdownload/file?token=6XIv5TUPto7pByrTQKhd6kwvyKLG2uY2wMMR8cNXqaA_Cv_isgaUtBzp4QtiozLO
基础支付 | H5支付 | H5下单 |  /v3/pay/transactions/h5
基础支付 | H5支付 | 查询订单_商户订单号查询 |  /v3/pay/transactions/out-trade-no/{{将此处替换为要查询的商户订单号}}?mchid={{mchid}}
基础支付 | H5支付 | 查询订单_微信支付订单号查询 |  /v3/pay/transactions/id/{{将此处替换为要查询的微信支付单号}}?mchid={{mchid}}
基础支付 | H5支付 | 关闭订单 |  /v3/pay/transactions/out-trade-no/{{此处替换为要关闭的商家订单号}}/close
基础支付 | H5支付 | 申请退款 |  /v3/refund/domestic/refunds
基础支付 | H5支付 | 查询单笔退款 |  /v3/refund/domestic/refunds/{{此处替换为要查询的商户自己生成的退款单号}}
基础支付 | H5支付 | 申请交易账单 |  /v3/bill/tradebill?bill_date=2022-12-31&bill_type=ALL
基础支付 | H5支付 | 申请资金账单 |  /v3/bill/fundflowbill?bill_date=2022-12-31
基础支付 | H5支付 | 下载账单 |  /v3/billdownload/file?token=6XIv5TUPto7pByrTQKhd6kwvyKLG2uY2wMMR8cNXqaA_Cv_isgaUtBzp4QtiozLO
基础支付 | Native支付 | Native下单 |  /v3/pay/transactions/native
基础支付 | Native支付 | 查询订单_商户订单号查询 |  /v3/pay/transactions/out-trade-no/{{将此处替换为要查询的商户订单号}}?mchid={{mchid}}
基础支付 | Native支付 | 查询订单_微信支付订单号查询 |  /v3/pay/transactions/id/{{将此处替换为要查询的微信支付单号}}?mchid={{mchid}}
基础支付 | Native支付 | 关闭订单 |  /v3/pay/transactions/out-trade-no/{{此处替换为要关闭的商家订单号}}/close
基础支付 | Native支付 | 申请退款 |  /v3/refund/domestic/refunds
基础支付 | Native支付 | 查询单笔退款 |  /v3/refund/domestic/refunds/{{此处替换为要查询的商户自己生成的退款单号}}
基础支付 | Native支付 | 申请交易账单 |  /v3/bill/tradebill?bill_date=2022-12-31&bill_type=ALL
基础支付 | Native支付 | 申请资金账单 |  /v3/bill/fundflowbill?bill_date=2022-12-31
基础支付 | Native支付 | 下载账单 |  /v3/billdownload/file?token=6XIv5TUPto7pByrTQKhd6kwvyKLG2uY2wMMR8cNXqaA_Cv_isgaUtBzp4QtiozLO
基础支付 | 小程序支付 | JSAPI下单 |  /v3/pay/transactions/jsapi
基础支付 | 小程序支付 | 查询订单_商户订单号查询 |  /v3/pay/transactions/out-trade-no/{{将此处替换为要查询的商户订单号}}?mchid={{mchid}}
基础支付 | 小程序支付 | 查询订单_微信支付订单号查询 |  /v3/pay/transactions/id/{{将此处替换为要查询的微信支付单号}}?mchid={{mchid}}
基础支付 | 小程序支付 | 关闭订单 |  /v3/pay/transactions/out-trade-no/{{此处替换为要关闭的商家订单号}}/close
基础支付 | 小程序支付 | 申请退款 |  /v3/refund/domestic/refunds
基础支付 | 小程序支付 | 查询单笔退款 |  /v3/refund/domestic/refunds/{{此处替换为要查询的商户自己生成的退款单号}}
基础支付 | 小程序支付 | 申请交易账单 |  /v3/bill/tradebill?bill_date=2022-12-31&bill_type=ALL
基础支付 | 小程序支付 | 申请资金账单 |  /v3/bill/fundflowbill?bill_date=2022-12-31
基础支付 | 小程序支付 | 下载账单 |  /v3/billdownload/file?token=6XIv5TUPto7pByrTQKhd6kwvyKLG2uY2wMMR8cNXqaA_Cv_isgaUtBzp4QtiozLO
经营能力 | 支付即服务 | 服务人员注册 |  /v3/smartguide/guides
经营能力 | 支付即服务 | 服务人员分配 |  /v3/smartguide/guides/{{请把此处替换服务人员ID}}/assign
经营能力 | 支付即服务 | 服务人员查询 |  /v3/smartguide/guides?store_id=1234
经营能力 | 支付即服务 | 服务人员信息更新 |  /v3/smartguide/guides/{{请将此处替换为服务人员ID}}
行业方案 | 智慧商圈 | 商圈会员积分同步 |  /v3/businesscircle/points/notify
行业方案 | 智慧商圈 | 商圈会员积分服务授权查询 |  /v3/businesscircle/user-authorizations/{{此处替换为要查询的openid}}?appid={{appid}}
行业方案 | 智慧商圈 | 商圈会员待积分状态查询 |  /v3/businesscircle/users/{{此处替换为要查询的openid}}/points/commit_status?brandid=1000&appid={{appid}}
行业方案 | 智慧商圈 | 商圈会员停车状态同步 |  /v3/businesscircle/parkings
行业方案 | 微信支付分停车服务 | 查询车牌服务开通信息 |  /v3/vehicle/parking/services/find?appid={{appid}}&plate_number=%E7%B2%A4B888888&plate_color=BLUE&openid={{openid}}
行业方案 | 微信支付分停车服务 | 创建停车入场 |  /v3/vehicle/parking/parkings
行业方案 | 微信支付分停车服务 | 扣费受理 |  /v3/vehicle/transactions/parking
行业方案 | 微信支付分停车服务 | 查询订单 |  /v3/vehicle/transactions/out-trade-no/{{此处替换为你要查询的订单号}}
行业方案 | 微信支付分停车服务 | 申请退款 |  /v3/refund/domestic/refunds
行业方案 | 微信支付分停车服务 | 查询单笔退款 |  /v3/refund/domestic/refunds/{{此处替换为要查询的退款单号}}
营销工具 | 代金券 | 创建代金券批次 |  /v3/marketing/favor/coupon-stocks
营销工具 | 代金券 | 激活代金券批次 |  /v3/marketing/favor/stocks/{{此处替换为要激活的代金券批次ID}}/start
营销工具 | 代金券 | 发放代金券批次 |  /v3/marketing/favor/users/{{此处替换为你要发放的openid}}/coupons
营销工具 | 代金券 | 暂停代金券批次 |  /v3/marketing/favor/stocks/{{替换为你要暂停的代金券批次id}}/pause
营销工具 | 代金券 | 重启代金券批次 |  /v3/marketing/favor/stocks/{{替换为你要重启的代金券批次id}}/restart
营销工具 | 代金券 | 条件查询批次列表 |  /v3/marketing/favor/stocks?offset=0&limit=10&stock_creator_mchid=9856888&create_start_time=2015-05-20T13%3A29%3A35%2B08%3A00&create_end_time=2015-05-20T19%3A29%3A35%2B08%3A00&status=running
营销工具 | 代金券 | 查询批次详情 |  /v3/marketing/favor/stocks/{{此处替换为要查询的代金券批次ID}}?stock_creator_mchid=123456
营销工具 | 代金券 | 查询代金券详情 |  /v3/marketing/favor/users/{{openid}}/coupons/{{此处替换为要查询的代金券id}}?appid=wx233544546545989
营销工具 | 代金券 | 查询代金券可用商户 |  /v3/marketing/favor/stocks/9865000/merchants?offset=0&limit=10&stock_creator_mchid={{mchid}}
营销工具 | 代金券 | 查询代金券可用单品 |  /v3/marketing/favor/stocks/9865000/items?offset=10&limit=10&stock_creator_mchid={{mchid}}
营销工具 | 代金券 | 根据商户号查用户的券 |  /v3/marketing/favor/users/{{此处替换为要查询的用户的openid}}/coupons?appid={{appid}}
营销工具 | 代金券 | 下载批次核销明细 |  /v3/marketing/favor/stocks/{stock_id}/use-flow
营销工具 | 代金券 | 下载批次退款明细 |  /v3/marketing/favor/stocks/{stock_id}/refund-flow
营销工具 | 代金券 | 设置消息通知地址 |  /v3/marketing/favor/callbacks
营销工具 | 商家券 | 创建商家券 |  /v3/marketing/busifavor/stocks
营销工具 | 商家券 | 查询商家券详情 |  /v3/marketing/busifavor/stocks/{{此处替换为要查询的商家去批次ID}}
营销工具 | 商家券 | 核销用户券 |  /v3/marketing/busifavor/coupons/use
营销工具 | 商家券 | 根据过滤条件查询用户券 |  /v3/marketing/busifavor/users/{{openid}}/coupons?appid=wx233544546545989&stock_id=9865000&coupon_state=USED&creator_merchant=1000000001&offset=0&limit=20
营销工具 | 商家券 | 查询用户单张券详情 |  /v3/marketing/busifavor/users/{{openid}}/coupons/{{此处替换为要查询的商家券code}}/appids/{{appid}}
营销工具 | 商家券 | 上传预存code |  /v3/marketing/busifavor/stocks/{stock_id}/couponcodes
营销工具 | 商家券 | 设置商家券事件通知地址 |  /v3/marketing/busifavor/callbacks
营销工具 | 商家券 | 查询商家券事件通知地址 |  /v3/marketing/busifavor/callbacks?mchid={{mchid}}
营销工具 | 商家券 | 关联订单信息 |  /v3/marketing/busifavor/coupons/associate
营销工具 | 商家券 | 取消关联订单信息 |  /v3/marketing/busifavor/coupons/disassociate
营销工具 | 商家券 | 修改批次预算 |  /v3/marketing/busifavor/stocks/{{此处替换为要修改预算的商家券批次号}}/budget
营销工具 | 商家券 | 修改商家券基本信息 |  /v3/marketing/busifavor/stocks/{stock_id}
营销工具 | 商家券 | 申请退券 |  /v3/marketing/busifavor/coupons/return
营销工具 | 商家券 | 使券失效 |  /v3/marketing/busifavor/coupons/deactivate
营销工具 | 商家券 | 营销补差付款 |  /v3/marketing/busifavor/subsidy/pay-receipts
营销工具 | 商家券 | 查询营销补差付款单详情 |  /v3/marketing/busifavor/subsidy/pay-receipts/{{此处替换为要查询的补差付款单号}}
营销工具 | 委托营销 | 建立合作关系 |  /v3/marketing/partnerships/build
营销工具 | 委托营销 | 查询合作关系列表 |  /v3/marketing/partnerships?authorized_data=%7b%22business_type%22%3a%22BUSIFAVOR_STOCK%22%7d&partner=%7b%22type%22%3a%22APPID%22%7d&offset=0&limit=5
营销工具 | 支付有礼 | 创建全场满额送活动 |  /v3/marketing/paygiftactivity/unique-threshold-activity
营销工具 | 支付有礼 | 查询活动详情接口 |  /v3/marketing/paygiftactivity/activities/{{此处替换为要查询的支付有礼活动ID}}
营销工具 | 支付有礼 | 查询活动发券商户号 |  /v3/marketing/paygiftactivity/activities/{{此处替换为要查询的支付有礼活动ID}}/merchants?offset=0&limit=20
营销工具 | 支付有礼 | 查询活动指定商品列表 |  /v3/marketing/paygiftactivity/activities/{{此处替换为要查询的支付有礼活动ID}}/goods?offset=0&limit=20
营销工具 | 支付有礼 | 终止活动 |  /v3/marketing/paygiftactivity/activities/{{此处替换为要终止的支付有礼活动id}}/terminate
营销工具 | 支付有礼 | 新增活动发券商户号 |  /v3/marketing/paygiftactivity/activities/{{此处替换为要增加的支付有礼活动ID}}/merchants/add
营销工具 | 支付有礼 | 获取支付有礼活动列表 |  /v3/marketing/paygiftactivity/activities?offset=0&limit=10&activity_status=ONGOING_ACT_STATUS
营销工具 | 支付有礼 | 删除活动发券商户号 |  /v3/marketing/paygiftactivity/activities/{{此处替换为要修改的支付有礼活动ID}}/merchants/delete
资金应用 | 商家转账到零钱 | 发起商家转账 |  /v3/transfer/batches
资金应用 | 商家转账到零钱 | 微信批次单号查询批次单 |  /v3/transfer/batches/batch-id/{{此处替换为要查询的微信批次单号}}?need_query_detail=true&detail_status=ALL&offset=0&limit=20
资金应用 | 商家转账到零钱 | 微信明细单号查询明细单 |  /v3/transfer/batches/batch-id/{{此处替换为要查询的微信批次单号}}/details/detail-id/{{此处要替换为要查询的微信明细单号}}
资金应用 | 商家转账到零钱 | 商家批次单号查询批次单 |  /v3/transfer/batches/out-batch-no/{{此处替换为要查询的微信批次单号}}?need_query_detail=true&detail_status=ALL&offset=0&limit=20
资金应用 | 商家转账到零钱 | 商家明细单号查询明细单 |  /v3/transfer/batches/out-batch-no/{{此处替换为要查询的商家批次单号}}/details/out-detail-no/{{此处要替换为要查询的商家明细单号}}
资金应用 | 商家转账到零钱 | 转账电子回单申请受理 |  /v3/transfer/bill-receipt
资金应用 | 商家转账到零钱 | 查询转账电子回单 |  /v3/transfer/bill-receipt/{{此处替换为要查询的商家批次单号}}
资金应用 | 商家转账到零钱 | 转账明细电子回单受理 |  /v3/transfer-detail/electronic-receipts
资金应用 | 商家转账到零钱 | 查询转账明细电子回单受理结果 |  /v3/transfer-detail/electronic-receipts?accept_type=BATCH_TRANSFER&out_batch_no=pay2022022222220222&out_detail_no=MX2022022200001
资金应用 | 商家转账到零钱 | 下载电子回单 |  /v3/billdownload/file?token=6XIv5TUPto7pByrTQKhd6kwvyKLG2uY2wMMR8cNXqaA_Cv_isgaUtBzp4QtiozLO
资金应用 | 分账 | 请求分账 |  /v3/profitsharing/orders
资金应用 | 分账 | 查询分账结果 |  /v3/profitsharing/orders/{{此处替换为要查询的商家分账单号}}?transaction_id=4208450740201411110007820472
资金应用 | 分账 | 请求分账回退 |  /v3/profitsharing/return-orders
资金应用 | 分账 | 查询分账回退结果 |  /v3/profitsharing/return-orders/{{此处替换为要查询的商家分账回退单号}}?&out_order_no=P2022111111111100011
资金应用 | 分账 | 解冻剩余资金 |  /v3/profitsharing/orders/unfreeze
资金应用 | 分账 | 查询剩余待分金额 |  /v3/profitsharing/transactions/{{请将此处替换为要查询剩余待分金额的微信支付订单号}}/amounts
资金应用 | 分账 | 添加分账接收方 |  /v3/profitsharing/receivers/add
资金应用 | 分账 | 删除分账接收方 |  /v3/profitsharing/receivers/delete
资金应用 | 分账 | 申请分账账单 |  /v3/profitsharing/bills?bill_date=2022-11-11
风险合规 | 消费者投诉2.0 | 查询投诉单列表 |  /v3/merchant-service/complaints-v2?limit=5&offset=10&begin_date=2022-11-01&end_date=2022-11-11&complainted_mchid={{mchid}}
风险合规 | 消费者投诉2.0 | 查询投诉单详情 |  /v3/merchant-service/complaints-v2/{{请将此处替换为要查询的投诉单对应的投诉单号}}
风险合规 | 消费者投诉2.0 | 查询投诉协商历史 |  /v3/merchant-service/complaints-v2/{{请将此处替换为要查询的投诉单对应的投诉单号}}/negotiation-historys?limit=50&offset=0
风险合规 | 消费者投诉2.0 | 创建投诉通知回调地址 |  /v3/merchant-service/complaint-notifications
风险合规 | 消费者投诉2.0 | 查询投诉通知回调地址 |  /v3/merchant-service/complaint-notifications
风险合规 | 消费者投诉2.0 | 更新投诉通知回调地址 |  /v3/merchant-service/complaint-notifications
风险合规 | 消费者投诉2.0 | 删除投诉通知回调地址 |  /v3/merchant-service/complaint-notifications
风险合规 | 消费者投诉2.0 | 回复用户 |  /v3/merchant-service/complaints-v2/{{请将此处替换为要回复的投诉单对应的投诉单号}}/response
风险合规 | 消费者投诉2.0 | 反馈处理完成 |  /v3/merchant-service/complaints-v2/{{此处替换为反馈处理完成的投诉单号}}/complete
风险合规 | 消费者投诉2.0 | 更新退款审批结果 |  /v3/merchant-service/complaints-v2/{{此处替换为投诉单对应的投诉单号}}/update-refund-progress

TheNorthMemory commented on wechatpay-apiv3.postman_collection.json:
```diff
@@ -120,163 +111,8994 @@
+							]
+						},
+						{
+							"name": "服务人员查询",
+							"request": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{server_url}}/v3/smartguide/guides?store_id=1234",
+									"host": [
+										"{{server_url}}"
+									],
+									"path": [
+										"v3",
+										"smartguide",
+										"guides"
+									],
+									"query": [
+										{
+											"key": "store_id",
+											"value": "1234",
+											"description": "门店在微信支付商户平台的唯一标识（查找路径：登录商户平台—>营销中心—>门店管理，若无门店则需先创建门店）"
+										}
+									]
+								}
+							},
```
`服务人员查询` 接口，入参如果带 加密`phone`，请求头需要带  `Wechatpay-Serial` 平台证书序列号 
详见: https://pay.weixin.qq.com/wiki/doc/apiv3/apis/chapter8_4_3.shtml

Memory-9527 replied:
此处已更新

Memory-9527 committed:
Update wechatpay-apiv3.postman_collection.json

修正个别请求方式错误

tpirc3 reviewed (approved):
👍🏻👍🏻👍🏻
